CHANGELOG.md

```diff
@@ -16,7 +16,9 @@ Valid subsections within a version are:
 
 ## Unreleased
 
-Things to be included in the next release go here.
+### Added
```

Comment:
```suggestion
Things to be included in the next release go here.

### Added
```

This is required for the release automation

Reply:
"Things.." text added back in.